test/ci/buildkite/ngtf-ie-ubuntu.yml

```diff
@@ -34,6 +34,18 @@
       queue: ${BUILDKITE_AGENT_META_DATA_QUEUE}
       name: ${BUILDKITE_AGENT_META_DATA_NAME}
 
+  - wait
+  - command: |
+      source /localdisk/buildkite/artifacts/$BUILDKITE_BUILD_ID/venv/bin/activate 
+      PYTEST_SKIPFILTERS=`pwd`/test/python/filters_IECPU.py PYTHONPATH=`pwd`:`pwd`/tools:`pwd`/examples:`pwd`/examples/mnist python3 test/ci/buildkite/test_runner.py \
```

Comment:
I'm wondering if it makes sense to use the same env-var for exclusion manifest for both gtests and pytests or at least normalize it so that it's easier to remember?

Reply:
In gtest case, the file allows both inclusion and exclusion patterns. In this case we just specify skipped ones. Also, mostly these will be used via CI which won't be changed frequently.

Reply:
Too many undocumented env vars to remember. Why don't we be consistent across the different tests we have?

Reply:
Named Bridge and TF pytest skip filter env vars similar: PYTEST_SKIPFILTERS and TFPYTEST_SKIPFILTERS

Reply:
Yeah, the naming can be improved but this is OK for now.